src/components/QRReader/QRReader.tsx

```diff
@@ -64,24 +64,48 @@ export const QRReader: FC<QRReaderProps> = ({ isEthereum, onScan }) => {
 };
 
 const QRReaderWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: calc(var(--gap) / 2);
+  section {
+    width: 362px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @media (max-width: 567px) {
+      width: 100%;
+    }
+  }
   div {
     height: 362px;
     padding-top: 0 !important;
     video {
-      height: unset !important;
+      // height: unset !important;
```

Comment:
test with collection "28" on opal pls (videos)

Reply:
this video item is only used for qr code reader